tests_integ/helpers/helpers.py

```diff
@@ -0,0 +1,147 @@
+import json
```

Comment:
Do we have a better name then "helpers"? We should try to be explicit (single principle) and in my experience, these types of classes/files just become dumping grounds

Reply:
Agree, also `helpers/helpers.py` is redundant.
We could maybe move the `transform_template` function to `helpers\template.py` and the rest to `helpers\resource.py`.

Reply:
Updated

Reply:
Updated